alicat_device:
  ros__parameters:
    flowrate_service: alicat_device/flowrate_service
    # device_port: /dev/ttyUSB0
    # device_serial_number: FT4U1ID2
    # device_serial_number: FT4TF9NT
    device_serial_number: FT4U1ID2

alicat_client:
  ros__parameters:
    flowrate_service: alicat_device/flowrate_service

flowrate_publisher:
  ros__parameters:
    flowrate_service: alicat_device/flowrate_service
